src/AccessibilityInsights.CI/AccessibilityInsights.CI.nuspec

```diff
@@ -1,18 +1,23 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <package>
   <metadata>
-    <id>AxeWindows</id>
+    <id>Axe.Windows</id>
     <version>$buildnumber$</version>
-    <title>AxeWindows Automation Library</title>
-    <summary>AxeWindows automation library</summary>
-    <description>This package provides the AxeWindows core library. This library can be accessed either via a .NET app, or from PowerShell via "Install-Module Axe.Windows.Automation.dll", followed by "Start-AxeWindows", "Invoke-Snapshot" and "Stop-AxeWindows"</description>
-    <authors>AxeWindows team</authors>
+    <title>axe-windows</title>
+    <summary>Automated accessibility analysis for Windows applications</summary>
+    <description>The package provides the assemblies necessary to automate accessibility testing for Windows applications using the included axe-windows rule set. Analysis can be run either via a .NET app, or from PowerShell via "Install-Module Axe.Windows.Automation.dll", followed by "Start-AxeWindows", "Invoke-Snapshot" and "Stop-AxeWindows"</description>
+    <projectUrl>https://github.com/Microsoft/axe-windows</projectUrl>
+    <iconUrl>https://github.com/Microsoft/axe-windows/raw/master/brand/brand-blue-128px.png</iconUrl>
+    <authors>Microsoft</authors>
     <owners></owners>
-    <copyright>© 2018. All rights reserved.</copyright>
-    <tags>AxeWindows Accessibility Automation</tags>
+    <license type="expression">MIT</license>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <language>en-US</language>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
```

Comment:
>Microsoft Corporation [](start = 17, length = 21)

At one point, we had "Microsoft" in the assembly copyrights, but we were explicitly asked to remove it on the fear that it would tend to discourage open source contributions. I'll see if I can dig up the email. (To be clear, this isn't a blocker for the PR and could be changed later if necessary)